### DISCUSSION TITLE
please build this

### DISCUSSION THREAD

**Discussion by sr71684** [4Source/draw-enable-obsidian-plugin#5] [General]
### Description of the unfavorable solution

This would be an epic feature if it were feature complete

### Describe the feature you would like to have

Wondering when it would be ready

### Additional context

**Comment by 4Source** [comment]
Unfortunately I'm very busy at the moment and don't have much time to work on it. It's also not easy to implement and I haven't seen anything even nearly similar for CodeMirror (which Obsidian is based on). I'm having trouble getting started because I'm having a hard time learning CodeMirror.

But I WILL work on it as soon as I can, I would also like the opportunity to use the planned features. You just have to wait a while.

If you have any ideas about important features, please feel free to share them with us.
